### PR TITLE
added insert operation in graphql

### PR DESCRIPTION
**Pull request by atharva29**

### CODE REVIEW

File: utils/graphql/delete.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+	deleteRequest.Find, err = extractWhereClause(field.Arguments, store)
+	if err != nil {
+		return nil, err
+	}
+
+	return &deleteRequest, nil
+}
+
+func extractWhereClause(args []*ast.Argument, store m) (m, error) {
```
Comment (YourTechBud) [inline utils/graphql/delete.go]: Why are you re declaring this function?

File: utils/graphql/update.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+	return m{"status": status}, graph.crud.Update(context.TODO(), dbType, graph.project, col, req)
+}
+
+func generateUpdateRequest(field *ast.Field, store m) (*model.UpdateRequest, error) {
+	var err error
+	var updateRequest model.UpdateRequest
+
+	//---	 upsert identification is wrong
+	if strings.HasPrefix(field.Name.Value, "upsert") {
```
Comment (YourTechBud) [inline utils/graphql/update.go]: This if else condition isn't needed! Instead make a `extractUpdateOperstion` function which returns the operation. The operation is available as the `op` field in the arguments.

File: utils/graphql/update.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+	updateRequest.Update, err = extractUpdateArgs(field.Arguments, store)
+	if err != nil {
+		return nil, err
+	}
+
+	return &updateRequest, nil
+}
+
+func extractWhereClause(args []*ast.Argument, store m) (m, error) {
```
Comment (YourTechBud) [inline utils/graphql/update.go]: Dont redeclare this function again

File: utils/graphql/update.go [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+		}
+	}
+	return m{}, nil
+}
+
+func extractUpdateArgs(args []*ast.Argument, store m) (m, error) {
+	var t map[string]interface{}
+	for _, v := range args {
+		switch v.Name.Value {
+		case "set", "inc", "mul", "max", "min", "currentTimestamp", "currentDate":
```
Comment (YourTechBud) [inline utils/graphql/update.go]: `push`, `rename`, `remove` are missing

Comment (atharva29) [reply utils/graphql/update.go]: ok

File: utils/graphql/update.go (hunk 3 above)
Comment (atharva29) [reply utils/graphql/update.go]: ok

**Pull request merged**